SUI/SUI/ContentView.swift

```diff
@@ -8,15 +8,120 @@
 import SwiftUI
 
 struct ContentView: View {
+    private enum Constants {
+        static let toggleText = "Показать настройки"
+        static let roundedRectangleCornerRadius = 10
+        static let someText = "Что-то на экране"
+        static let isXOffsetChange = 100
+        static let originalXOffset = 0
+        static let firstWalletText = "Кошелек 1"
+        static let secondWalletText = "Кошелек 2"
+        static let profileText = "Профиль"
+        static let spacerHeight = 400
+        static let response = 0.5
+        static let dampingFraction = 0.7
+        static let blendDuration = 0.3
+        static let cashInButtonName = "Пополнить баланс"
+        static let openAnAccountButtonName = "Открыть счет"
+        static let openAnAccountAlertTitle = "Открыть счет?"
+        static let transferButtonName = "Перевести со счета на счет"
+        static let replenishButtonName = "Пополнить"
+        static let balanceConfDialogText = "Пополнить счет?"
+        static let primaryButtonOpenAccountText = "Пополнить"
+        static let transferAlertTitle = "Перевести 300 у.е с кошелька 1 на кошелек 2?"
+        static let transferButtonTitle = "Перевести"
+        static let zeroOpacity = 0
+        static let oneOpacity = 1
+        static let balance = 300
+    }
+    
+    @State var isOnToggle = false
+    @State var isBalanceButtonIsTapped = false
+    @State var isOpenAnAccountButtonTapped = false
+    @State var isOpenedAccount = false
+    @State var isTransferButtonTapped = false
+    @State var firstWalletBalance = 0
+    @State var secondWalletBalance = 0
+    @State var isReplenish = false
+    
     var body: some View {
         VStack {
-            Image(systemName: "globe")
-                .imageScale(.large)
-                .foregroundColor(.accentColor)
-            Text("Hello, world!")
-        }
+            ZStack {
+                HStack {
+                    VStack {
+                        Text(Constants.firstWalletText)
+                        Text("\(firstWalletBalance)")
+                        if !isOpenedAccount {
+                            Text(Constants.secondWalletText).opacity(Double(Constants.zeroOpacity))
+                            Text("\(secondWalletBalance)").opacity(Double(Constants.zeroOpacity)) } else {
+                                Text(Constants.secondWalletText).opacity(Double(Constants.oneOpacity))
+                                Text("\(secondWalletBalance)").opacity(Double(Constants.oneOpacity))
+                            }
+                        Text(Constants.profileText)
+                        Spacer().frame(height: CGFloat(Constants.spacerHeight))
+                    }
+                    Spacer()
+                }
+                Spacer()
+                RoundedRectangle(cornerRadius: CGFloat(Constants.roundedRectangleCornerRadius)).fill(.yellow).offset(x: isOnToggle ? CGFloat(Constants.isXOffsetChange): CGFloat(Constants.originalXOffset))
```

Comment:
```suggestion
                RoundedRectangle(cornerRadius: CGFloat(Constants.roundedRectangleCornerRadius)).fill(.yellow).offset(x: isOnToggle ? CGFloat(Constants.isXOffsetChange): CGFloat(Constants.originalXOffset))
```
форматирование! нажимай ввод перед точкой